docs/api/rest_api/rest_api_reference/input/ez-types.raml

```diff
@@ -1071,6 +1081,7 @@ Message:
         values:
             description: 'Translation value objects. May not contain any numbers, which might result in requiring plural forms. Use Plural for that.'
             type: array
+            items: object
```

Comment:
This has been added to avoid `PHP Deprecated:  mb_substr(): Passing null to parameter #1 ($string) of type string is deprecated in tools/raml2html/vendor/raml-org/raml-php-parser/src/Type.php on line 85`

While trying to better define what those item objects are, I discovered that the type `Message` is in fact never used. If `Message` was used, removing it would produce `An exception has been thrown during the rendering of a template ("No type found for name 'Message'").` which is not the case.

I wrote a script testing if types defined in ez-types.raml are used and it found several unused ones.

I think that this is not the PR to use or to remove unused types. I stay with the minimal fix.

Reply:
See #2042